src/chainstate/stacks/db/blocks.rs

```diff
@@ -4599,6 +4601,8 @@ impl StacksChainState {
                                    "txid" => %txid,
                                    "burn_block" => %burn_header_hash,
                                    "contract_call_ecode" => %resp.data);
+                        } else {
+                            debug!("Processed StackStx burn op for {} uSTX for {} cycles starting at {} from {} to pay out to {}", stacked_ustx, num_cycles, block_height, &sender, &reward_addr);
```

Comment:
All of those fields should be k-v entries:

```suggestion
                            debug!("Processed StackStx burnchain op"; "amount_ustx" => stacked_ustx, "num_cycles" => num_cycles, "burn_block_height" => block_height, "sender" => %sender, "reward_addr" => %reward_addr);
```

Reply:
Would be useful to add the `txid` as well